src/main/default/classes/RatingsHandler.cls

```diff
@@ -1,11 +1,15 @@
 public with sharing class RatingsHandler {
     @AuraEnabled(cacheable=false)
-    public static List<Rating__c> getRating(String documentId) {
-        return [
+    public static Rating__c getRating(String documentId) {
+        List<Rating__c> records = [
             SELECT Score__c, Document_id__c
             FROM Rating__c
             WHERE Document_id__c = :documentId
             LIMIT 1
         ];
+        if (records.size() > 0) {
+            return records[0];
+        }
+        return null;
```

Comment:
Would it be possible to return a default rating object so the caller does not have to worry with `null` ratings?

Reply:
Actually the function calling this method needs to know that a rating was not found and that's why I'm returning null here.  the reason why this caller function needs the information that a rating wasn't found is that not finding a rating is the trigger to create  a new rating object for this document suggestion.
The body of the  function calling this method is something like that:
```
const rating = getRating(DOCUMENT_ID)
 if(!rating){
     // this document suggestion doesn't have a rating object yet, it's the first time a rating object is needed for this document so let's create it
    createRating(DOCUMENT_ID)
   return 0;// the initial score
}
return rating.score
```

Reply:
Ok, I got confused between the meaning of `rating` and `score`, I guess. Yeah, it is OK to have `getRating` return `null` when the record does not exist.